backend/src/infrastructure/schema/user_schema.py

```
@@ -0,0 +1,12 @@
+from cryptography.fernet import Fernet
```

Ruff (0.12.2)

`cryptography.fernet.Fernet` imported but unused

Remove unused import: `cryptography.fernet.Fernet`

(F401)